driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -200,6 +200,9 @@ public class Client {
     /* note this must end with '=' */
     private final String SESSION_COOKIE_FIELD = "session=";
 
+    /* header returned from proxy if request was delayed */
+    private final String X_RATE_LIMIT_DELAYED = "X-NoSQL-RLDelay-Ms";
```

Comment:
Is there a reason you didn't add this constant to util/HttpConstants? The name is also a bit awkward. Given that header names are case-insensitive why not something like x-nosql-delay-ms? Is the "rl" necessary?